front/tests/pages/common-page-model.ts

```diff
@@ -38,6 +41,15 @@ class CommonPage {
       await this.viteOverlay.evaluate((node) => node.setAttribute('style', 'display:none;'));
     }
   }
+
+  async closeToastNotification(): Promise<void> {
+    const toastCount = await this.toastSNCF.count();
+
+    if (toastCount > 0) {
```

Comment:
We don't need to check the length here. We can just call `.all()` always (which will return an empty array if there are no matching elements).

Reply:
There are no unnecessary operations performed when the list is empty. On the contrary, the current code performs two queries against the browser instead of a single one.

Reply:
Done